torch/ao/quantization/fx/match_utils.py

```diff
@@ -22,17 +23,49 @@
 MatchResult = Tuple[Node, List[Node], Optional[Pattern], QuantizeHandler,
                     QConfigAny]
 
+# Module types in this list are allowed to be used by multiple 'call_module'
+# nodes. Module types not in this list will not be matched if they are used by
+# multiple 'call_module' nodes.
+MODULE_TYPES_SAFE_TO_USE_BY_MULTIPLE_NODES = set([
+    torch.nn.ReLU,
```

Comment:
I remember Kimish reported another case, there is a Lienar - relu pattern, and then then people access the bias in linear, should we add Linear/Conv here as well?

Reply:
Oh sorry, this is safe to use, sounds good